integration

```diff
@@ -167,6 +167,7 @@
   "benjamin-dcs/file-plusplus",
   "benleb/sureha",
   "Bennett-Wendorf/hass-uplift-desk",
+  "BenoitDuffez/home-assistant",
```

Comment:
This repository does not exist:

https://github.com/BenoitDuffez/home-assistant

../Frenck